crates/e2e/src/setup/services.rs

```diff
@@ -88,7 +88,7 @@ impl<'a> Services<'a> {
             "--auction-update-interval=1".to_string(),
             format!("--ethflow-contract={:?}", self.contracts.ethflow.address()),
             "--skip-event-sync=true".to_string(),
-            "--solve-deadline=2".to_string(),
+            "--solve-deadline=6".to_string(),
```

Comment:
Having to adjust this value seems to be another indicator that reading those absolute values from a a config file might not be the best way to do it.

Reply:
Can this change be reverted now?